modules/sdk/gradle-plugins-workspace/src/main/resources/schemas/client-extension.schema.json

```diff
@@ -181,6 +181,14 @@
 					],
 					"description": "Description for customElement"
 				},
+				"editorConfigContributor": {
+					"allOf": [
+						{
+							"$ref": "#/definitions/propertySets/properties/url"
+						}
+					],
+					"description": "Description for editorConfigContributor"
```

Comment:
@markocikos is it reasonable to add a basic description for this one? The rest are placeholders from the first bulk add, but going forward each product team should include as much info as possible with each contribution, since they understand each individual type. I will do my best to keep verbiage and language consistent across types.

Reply:
@drewbrokke I adding a description that is a copy of description from UI.

Reply:
Great, thank you @markocikos